doc-examples/src/main/java/csvsamples/FetchHttpEvery30SecondsAndConvertCsvToJsonToKafkaInJava.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package csvsamples;
+
+// #sample
+                import akka.Done;
+                import akka.actor.ActorSystem;
+                import akka.actor.Cancellable;
+                import akka.http.javadsl.Http;
+                import akka.http.javadsl.model.HttpRequest;
+                import akka.http.javadsl.model.HttpResponse;
+                import akka.http.javadsl.model.MediaRanges;
+                import akka.http.javadsl.model.StatusCodes;
+                import akka.http.javadsl.model.headers.Accept;
+                import akka.japi.Pair;
+                import akka.kafka.ConsumerSettings;
+                import akka.kafka.ProducerSettings;
+                import akka.kafka.Subscriptions;
+                import akka.kafka.javadsl.Consumer;
+                import akka.kafka.javadsl.Producer;
+                import akka.stream.ActorMaterializer;
+                import akka.stream.Materializer;
+                import akka.stream.alpakka.csv.javadsl.CsvParsing;
+                import akka.stream.alpakka.csv.javadsl.CsvToMap;
+                import akka.stream.javadsl.Keep;
+                import akka.stream.javadsl.Sink;
+                import akka.stream.javadsl.Source;
+                import akka.util.ByteString;
+                import com.fasterxml.jackson.core.JsonFactory;
+                import com.fasterxml.jackson.core.JsonGenerator;
+                import org.apache.kafka.clients.consumer.ConsumerConfig;
+                import org.apache.kafka.clients.consumer.ConsumerRecord;
+                import org.apache.kafka.clients.producer.ProducerRecord;
+                import org.apache.kafka.common.serialization.StringDeserializer;
+                import org.apache.kafka.common.serialization.StringSerializer;
+                import playground.KafkaEmbedded;
+
+                import java.io.IOException;
+                import java.io.StringWriter;
+                import java.time.Duration;
+                import java.util.Collections;
+                import java.util.HashMap;
+                import java.util.Map;
+                import java.util.concurrent.CompletionStage;
+                import java.util.concurrent.Executor;
+                import java.util.concurrent.Executors;
+
+// #sample
+
+public class FetchHttpEvery30SecondsAndConvertCsvToJsonToKafkaInJava {
+
+    public static void main(String[] args) throws Exception {
```

Comment:
We have mixed indentation across the files. Some use two spaces and this among others use four. I think for inclusion to documentation as snippets and in general two spaces looks better. :)